modular_zubbers/code/modules/vending/games.dm

```diff
@@ -4,117 +4,121 @@
 			"name" = "Toys",
 			"icon" = "hat-wizard",
 			"products" = list(
-				/obj/item/toy/plush/skyrat/fox/mia = 3,
-				/obj/item/toy/plush/skyrat/fox/kailyn = 3,
-				/obj/item/toy/plush/skyrat/voodoo = 5,
-				/obj/item/toy/plush/skyrat/immovable_rod = 3,
-				/obj/item/toy/plush/skyrat/tesh = 3,
-				/obj/item/toy/plush/skyrat/roselia = 3,
-				/obj/item/toy/plush/skyrat/chunko/plushie_winrow = 3,
-				/obj/item/toy/plush/skyrat/chunko/bonnie = 3,
-				/obj/item/toy/plush/skyrat/chunko/andrew = 3,
-				/obj/item/toy/plush/skyrat/chunko/inessa = 3,
-				/obj/item/toy/plush/skyrat/plushie_star = 3,
-				/obj/item/toy/plush/skyrat/plushie_chiara = 3,
-				/obj/item/toy/plush/skyrat/plushie_dan = 3,
-				/obj/item/toy/plush/skyrat/fox/plushie_jeanne = 3,
-				/obj/item/toy/plush/skyrat/plushie_azyre = 3,
-				/obj/item/toy/plush/skyrat/plushie_razurath = 3,
-				/obj/item/toy/plush/skyrat/plushie_razurath/second = 3,
-				/obj/item/toy/plush/skyrat/plushie_elofy = 3,
-				/obj/item/toy/plush/skyrat/plushie_syntax1112 = 3,
-				/obj/item/toy/plush/moth/lovers = 3,
-				/obj/item/toy/plush/goatplushie = 3,
-				/obj/item/toy/plush/sunny_plush = 3,
-				/obj/item/toy/plush/chirp_plush = 3,
+				/obj/item/laser_pointer/limited = 3,
+				/obj/item/storage/briefcase/secure/white/wargame_kit = 3,
+				/obj/item/toy/plush/abductor = 3,
```

Comment:
I feel like for plushies in particular, maybe down the line, it could be a typesof list instead. It seems tedious to keep a large list of plush subtypes assigned like this.